backend/src/services/enhanced_weather_service.py

```
@@ -325,7 +361,9 @@ async def get_weather_for_dates(
             }
             
         except Exception as e:
-            logger.error(f"Failed to get weather for date range {start_date} to {end_date}: {e}")
+            logger.error(
+                f"Failed to get weather for date range {start_date} to {end_date}: {e}"
+            )
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)